src/github/client/github-client.ts

```diff
@@ -99,10 +99,11 @@ export default class GitHubClient {
 		return response;
 	}
 
-	private async graphql<T>(query: string): Promise<AxiosResponse> {
-		return await this.axios.post<T>("https://api.github.com/graphql",
+	private async graphql<T>(query: string, variables?: Record<string, string | number | undefined>): Promise<AxiosResponse<{data: T}>> {
+		return  await this.axios.post<{data: T}>("https://api.github.com/graphql",
```

Comment:
instead of `<{data: T}>` you should use `<GraphQlQueryResponse<T>>` as this is the github "standard" which can also return errors